catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -1,487 +0,0 @@
-import ddf.catalog.content.operation.CreateStorageRequest;
-import ddf.catalog.content.operation.impl.CreateStorageResponseImpl;
-import ddf.catalog.content.operation.impl.DeleteStorageResponseImpl;
-import ddf.catalog.content.operation.impl.ReadStorageRequestImpl;
-import ddf.catalog.content.operation.impl.ReadStorageResponseImpl;
-import ddf.catalog.content.operation.impl.UpdateStorageResponseImpl;
-import ddf.catalog.data.AttributeRegistry;
-import ddf.catalog.data.Metacard;
-import ddf.catalog.data.impl.AttributeImpl;
-import ddf.catalog.operation.CreateRequest;
-import ddf.catalog.operation.CreateResponse;
-import ddf.catalog.operation.SourceInfoResponse;
-import ddf.catalog.operation.impl.OperationImpl;
-import ddf.catalog.operation.impl.ResponseImpl;
-import ddf.catalog.source.SourceDescriptor;
-import ddf.security.service.SecurityManager;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.TimeUnit;
-import javax.inject.Inject;
-import org.apache.camel.CamelContext;
-import org.apache.commons.io.IOUtils;
-import org.codice.ddf.catalog.content.impl.FileSystemStorageProvider;
-import org.codice.ddf.catalog.content.monitor.configurators.KeystoreTruststoreConfigurator;
-import org.codice.ddf.catalog.content.monitor.features.CamelFeatures;
-import org.codice.ddf.catalog.content.monitor.features.CxfFeatures;
-import org.codice.ddf.catalog.content.monitor.features.KarafSpringFeatures;
-import org.codice.ddf.catalog.content.monitor.features.SecurityFeatures;
-import org.codice.ddf.catalog.content.monitor.features.TinkFeatures;
-import org.codice.ddf.catalog.content.monitor.util.BundleInfo;
-import org.codice.ddf.test.common.ComponentTestRunner;
-import org.codice.ddf.test.common.annotations.AfterExam;
-import org.codice.ddf.test.common.annotations.BeforeExam;
-import org.codice.ddf.test.common.annotations.PaxExamRule;
-import org.codice.ddf.test.common.configurators.PortFinder;
-import org.glassfish.grizzly.http.Method;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
-import org.junit.runner.RunWith;
-import org.mockito.ArgumentCaptor;
-import org.mockito.stubbing.Answer;
-import org.ops4j.io.FileUtils;
-import org.ops4j.pax.exam.Option;
-import org.ops4j.pax.exam.karaf.options.KarafDistributionOption;
-import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
-import org.ops4j.pax.exam.spi.reactors.PerClass;
-import org.ops4j.pax.tinybundles.core.TinyBundles;
-
-@RunWith(ComponentTestRunner.class)
-@ExamReactorStrategy(PerClass.class)
-public class ContentDirectoryMonitorIT extends AbstractComponentTest {
```

Comment:
Because it was hopelessly broken with this change. It would need to be built using everything that is done to run the itests, at which point we can just run the itests.

Reply:
Does this mean we won't be able to have component tests?